tests/Microsoft.DotNet.Framework.Docker.Tests/SdkOnlyImageTests.cs

```diff
@@ -76,5 +77,26 @@ public void VerifyTargetingPacks(ImageDescriptor imageDescriptor)
 
             Assert.Equal(expectedVersions, actualVersions);
         }
+
+        [SkippableTheory("4.6.2", "4.7", "4.7.1", "4.7.2")]
```

Comment:
Should the `TraitAttribute` be added here?  IIRC this allows the caller to early run the tests for a single image variant.

Reply:
Yep, missed that.